Add Spanish translation - 'Optimization Workflow' section

moncho opened the pull request.

moncho committed:
Translate How to Optimize section to Spanish

0xjuanma commented on performance-es.md:
`s/paso/pasos`

0xjuanma commented on performance-es.md:
`s/Asegurate/Asegúrate`

0xjuanma commented on performance-es.md:
`s/mas/más`

0xjuanma commented on performance-es.md:
`s/Escribe código para el que puedas hacer benchmark/Escribe código que le puedes puedas aplicar benchmarks`

moncho replied:
Good suggestion!

0xjuanma commented on performance-es.md:
`s/Asegúrate/Asegúrate`

0xjuanma commented:
Looks great! Thanks! Just added a few comments. 👍

moncho committed:
Apply review suggestions and a few changes in the translation

for *profiling* and derivates, it was not always being translated
(as perfilado). I am not a great fan of the translation for this
term, but I think it looks better to keep the text consistent.

0xjuanma reviewed (approved):
👍

dgryski commented:
Is this ready to merge?

moncho commented:
I think it is, @dgryski
